src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.ChatBotCommandException;
+import duke.exception.ChatBotParameterException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.io.IOException;
```

Comment:
I believe the import statement for 'java.io.IOException' is in the wrong order, it should be before the package imports according to the AB3 checkstyle.

Reply:
Noted with thanks